source/js/components/join/join.jsx

```diff
@@ -202,63 +280,22 @@ export default class JoinUs extends React.Component {
     });
 
     return (
-      <div className="col-12 col-md-6 join-form">
-        <form onSubmit={evt => this.processFormData(evt)}>
-          <div className={inputGroupClass}>
-            <div className="mb-2">
-              <input
-                type="email"
-                className="form-control"
-                placeholder="EMAIL ADDRESS"
-                ref={el => (this.email = el)}
-                onFocus={evt => this.onInputFocus(evt)}
-              />
-            </div>
-            {this.state.apiSubmitted && !this.email.value && (
-              <small className="form-check form-control-feedback">
-                Please enter your email
-              </small>
-            )}
-            {this.state.signupFailed && (
-              <small className="form-check form-control-feedback">
-                Something went wrong. Please check your email address and try
-                again
-              </small>
-            )}
-          </div>
-          <div className={privacyClass}>
-            <label className="form-check-label mb-4">
-              <input
-                type="checkbox"
-                className="form-check-input"
-                id="PrivacyCheckbox"
-                ref={el => (this.privacy = el)}
-              />
-              <span className="form-text">
-                I'm okay with Mozilla handling my info as explained in this{" "}
-                <a href="https://www.mozilla.org/privacy/websites/">
-                  Privacy Notice
-                </a>
-              </span>
-              {this.state.apiSubmitted && !this.privacy.checked && (
-                <small className="has-danger">
-                  Please check this box if you want to proceed
-                </small>
-              )}
-            </label>
-            <div>
-              <button className="btn btn-normal join-btn">Sign Up</button>
-            </div>
+      <div className="col-12">
+        <form onSubmit={evt => this.processFormData(evt)} className="row">
+          <div className={fieldsWrapperClass}>
+            {this.renderEmailField(inputGroupClass)}
```

Comment:
we don't use these `inputGroupClass` and `privacyClass` in this function, so let's not compute them here, but instead move them to the functions that actually need them.